kubernetes/crds/machine.sapcloud.io_machinedeployments.yaml

```diff
@@ -59,9 +59,6 @@ spec:
             object represents. Servers may infer this from the endpoint the client
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
-        metadata:
```

Comment:
I hope this change is intensional?

Reply:
Yes. Prashanth. It is the solution for the issue - https://github.com/gardener/machine-controller-manager/pull/551

Reply:
Okay. In future, let's try to move minor independent fixes into it's on PRs. Just to keep it followable via commits.